butterfree/migrations/database_migration/database_migration.py

```diff
@@ -36,27 +41,40 @@ def _validate_schema(
             db_schema: object that contains the table og a given db schema.
 
         """
+        mismatches = []
+
+        if not db_schema:
+            return fs_schema
+
+        diff_list = list(filterfalse(lambda x: x in db_schema, fs_schema))
+
+        for feature in diff_list:
+            matching_features = [
+                x for x in db_schema if x["column_name"] == feature["column_name"]
+            ]
 
-    def _get_schema(self, db_client: Callable, table_name: str) -> List[Dict[str, Any]]:
+            if matching_features:
+                raise ValueError(f"The {feature['column_name']} can't be changed.")
```

Comment:
Why it can't be changed? If there is a reason, I would like to know more and maybe the user would like to know (while the error message won't help him/her) :)

Reply:
Also, if the automatic migration can handle it, it will break? Then the build will break?

Reply:
I've proposed some changes to how we'll deal with this case, please let me know your opinions w.r.t. this topic.